stores/libsql/src/vector/index.test.ts

```diff
@@ -254,7 +255,7 @@ describe('LibSQLVector', () => {
         const metadata = [
           { type: 'a', value: 1 },
           { type: 'b', value: 2 },
-          { type: 'a', value: 3 },
```

Comment:
with default minScore -1 this was now returning both `a` matches, changed it to `c` so the test assertions stay the same